include/fastdds/rtps/common/Locator.h

```diff
@@ -576,6 +594,8 @@ class LocatorList_t
         }
     }
 
+    FASTDDS_DEPRECATED_UNTIL(3, "eprosima::fastrtps::rtps::LocatorList_t::contains(const Locator_t&)",
+            "Not specify function with unknown functionality.")
```

Comment:
I am not sure about the reason of the deprecation and I do not think the user will understand it. Maybe it can be more clear and brief: "Unused method".